closed/src/java.base/share/classes/openj9/internal/security/RestrictedSecurityConfigurator.java

```diff
@@ -86,14 +100,24 @@ private RestrictedSecurityConfigurator() {
 
     /**
      * Restricted security mode will be enabled only if the semeru.fips system
-     * property is true (default as false).
+     * property is true (default as false), or semeru.restrictedsecurity is set,
+     * or CRIU checkpoint mode is enabled.
      *
      * @return true if restricted security is enabled
      */
     public static boolean isEnabled() {
         return securityEnabled;
     }
 
+/*[IF CRIU_SUPPORT]*/
+    /**
+     * Disables the restricted security mode.
+     */
+    public static void disable() {
```

Comment:
Is this to be invoked after CRIU restore?

Reply:
Yes, it is invoked after restore in the corresponding openj9 PR.

Reply:
Do you have an openj9 wip pr, or code snippet invoking this `disable()`?

Reply:
nv, it is https://github.com/eclipse-openj9/openj9/pull/16314

Reply:
Yes, it was linked above [eclipse-openj9/openj9#16314](https://github.com/eclipse-openj9/openj9/pull/16314)